### ISSUE TITLE
Uncaught SyntaxError: Unexpected token ')'

### ISSUE THREAD

**Issue by ivadenis** [newrelic/gatsby-plugin-newrelic#9]
[NOTE]: # ( ^^ Provide a general summary of the issue in the title above. ^^ )

## Description

Getting `Uncaught SyntaxError: Unexpected token ')'` after setting up the plugin.
Here is my config

```
// gatsby-config.js
{
  resolve: 'gatsby-plugin-newrelic',
  options: {
    instrumentationType: 'proAndSPA', // Options are 'lite', 'pro', 'proAndSPA'
    configs: {
      development: {
        accountId: 'xxxxx',
        trustKey: 'xxxxx',
        agentID: 'xxxxx',
        licenseKey: 'xxxxx',
        applicationID: 'xxxxx',
        beacon: 'bam.nr-data.net',
        errorBeacon: 'bam.nr-data.net',
      },
    },
  },
}
```

```
#.env
GATSBY_NEWRELIC_ENV='development'
```

<img width="840" alt="Screen Shot 2020-09-10 at 12 30 57 AM" src="https://user-images.githubusercontent.com/3453508/92695175-f556ac80-f2fc-11ea-8a9f-30d7d410dd51.png">

**Comment by passion4code** [comment]
yea, this is still waiting on #8 to get published as version 1.0.3

**Comment by devfreddy** [comment]
Published 🚀
